### DISCUSSION TITLE
arcaea server domain

### DISCUSSION THREAD

**Discussion by D-06** [Lost-MSth/Arcaea-server#40] [Q&A]
Anyone know where is arcaea server domain

**Comment by Lost-MSth** [comment]
I am not sure if "domain" you said means the host name.  
In `setting.py` you can set your server's host and port.

**Comment by D-06** [comment]
I mean the official arcaea server domain

**Reply by Lost-MSth** [reply]
Hmmm? There is nothing official in this project.

**Reply by D-06** [reply]
I mean the domain or just ip from lowiro use to run the game like lowiro.com or something similar

**Reply by Lost-MSth** [reply]
Why we have this?

**Reply by D-06** [reply]
Sorry my english is not very good

**Reply by D-06** [reply]
![Screenshot_20220211-155541_VNC Viewer.jpg](https://user-images.githubusercontent.com/82329926/153562763-b561e164-0698-406c-b488-9c8dd0d77cbb.jpg)

**Reply by D-06** [reply]
Did i missing something?

**Reply by Lost-MSth** [reply]
Emmm... If you use the client under v3.6.0, that's right.

**Reply by D-06** [reply]
I use 3.11.2 so it seem doesn't work

**Reply by D-06** [reply]
But for some reason arcaea server ip doesn't show up eventhough i use sslunpin or sslkiller

**Reply by D-06** [reply]
![Screenshot_20220211-160739_VNC Viewer.jpg](https://user-images.githubusercontent.com/82329926/153564308-1e636823-cc08-48e5-8e8d-0d40f58ef911.jpg)

**Reply by Lost-MSth** [reply]
I see the screenshot and it means you successed in remote mapping, so why the connect was down.

**Reply by D-06** [reply]
I don't know i expect the game to offline but i still can login the game like normal

**Reply by Lost-MSth** [reply]
You say that you can login in the official server and at the same time see something in Charles, right?
That's so strange.

**Reply by D-06** [reply]
Even i still can play like normal
![Screenshot_20220211-162040_VNC Viewer.jpg](https://user-images.githubusercontent.com/82329926/153566186-ea851c30-4930-44f1-9f5e-d5093718eb9d.jpg)
![Screenshot_20220211-162021_VMOS Pro.jpg](https://user-images.githubusercontent.com/82329926/153566224-189801c6-9f71-4a80-b35c-9a28ef600fd2.jpg)

**Reply by D-06** [reply]
> You say that you can login in the official server and at the same time see something in Charles, right?
> That's so strange.

Really strange 
or just the vm or something else

**Reply by Lost-MSth** [reply]
Oh, wait. You set too many map remote rules. Only 'arcapi-v2.lowiro.com' is needed.

**Reply by D-06** [reply]
![Screenshot_20220211-162743_VMOS Pro.jpg](https://user-images.githubusercontent.com/82329926/153567437-55acc986-4776-48ce-8b38-ddcc782bd85b.jpg)
Nope i still can login like normal

**Reply by Lost-MSth** [reply]
Hmmm maybe you need a proxy app.
> https://github.com/Lost-MSth/Arcaea-server/issues/20

**Reply by D-06** [reply]
I think i should change back to version 3.6.0 because even proxy app doesn't work also the bizarre thing in this version is charles doesn't seem to detect any domain related to arcaea like arcapi or arcapi-v2

**Reply by D-06** [reply]
Wait i remember in charles proxy website metion that android N or above is need to add modify some .xml file

**Reply by D-06** [reply]
Nevermind it doesn't work

**Comment by D-06** [comment]
Or just arcaea server ip

**Comment by MBRjun** [comment]
arcapi-v2.lowiro.com (54.214.148.207)
